lib/task_helpers/exports.rb

```diff
@@ -0,0 +1,46 @@
+module TaskHelpers
+  class Exports
+    class ExportsError < StandardError; end
+
+    # Illegal characters: '/', '|'
+    REGEX_KEEP_SPACES   = %r{[/|]}
+
+    # Illegal characters: '/', '|', ' '
+    REGEX_REMOVE_SPACES = %r{[/| ]}
+
+    # Replacement character
+    REPLACEMENT         = '_'.freeze
+
+    def self.safe_filename(filename, keep_spaces = false)
+      if keep_spaces
+        filename.gsub(REGEX_KEEP_SPACES, REPLACEMENT)
+      else
+        filename.gsub(REGEX_REMOVE_SPACES, REPLACEMENT)
+      end
+    end
+
+    def self.parse_options
+      require 'trollop'
+      options = Trollop.options(EvmRakeHelper.extract_command_options) do
+        opt :keep_spaces, 'Keep spaces in filenames', :type => :boolean, :short => 's', :default => false
+        opt :directory, 'Directory to place exported files in', :type => :string, :required => true
+      end
+
+      options
+    end
+
+    def self.validate_directory(directory)
```

Comment:
I think all of this should be done right after parsing the options.

Also this might work better if it just returned the string then you could do something like:

```ruby
error = validate_directory(options[:directory])
Trollop.die :directory, error if error
```

Then you don't have to write to standard out and you also don't need to define the exceptions.